terraform/product/simple_deployment/main.tf

```diff
@@ -0,0 +1,147 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+#--------------------------------------------------------
+# 1. DEPLOYMENTS
+#--------------------------------------------------------
+
+# main opensearch app
+module "opensearch" {
+  source = "../../charm/simple_deployment"
+
+  channel  = var.opensearch.channel
+  revision = var.opensearch.revision
+  base     = var.opensearch.base
+
+  app_name          = var.opensearch.app_name
+  units             = var.opensearch.units
+  config            = merge(var.opensearch.config, { "init_hold" : "false" })
+  model             = var.opensearch.model
+  constraints       = var.opensearch.constraints
+  storage           = var.opensearch.storage
+  endpoint_bindings = var.opensearch.endpoint_bindings
+}
+
+# OpenSearch dashboards
+module "opensearch-dashboards" {
+  source = "git::https://github.com/canonical/opensearch-dashboards-operator//terraform?ref=tf"
```

Comment:
I cannot find the `ref=tf` in this repo. I believe that will then default to the main branch's TF module (`2/edge`), right? In this case, we are leaving this ref as an optional?

Reply:
The `ref=tf` here is a reference to the remote git branch of the dashboards TF PR I made a few days back. This now should be changed to `ref=2/edge`.